File: 349-intersection-of-two-arrays/intersection-of-two-arrays.py
class Solution(object):
    def intersection(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[int]
        """
        list = [];
        for i in nums1:
            if (i in nums2) and (i not in list):
                list.append(i);

        return list;   
        

File: 349-intersection-of-two-arrays/intersection-of-two-arrays.java
class Solution {
    public int[] intersection(int[] nums1, int[] nums2) {
        HashSet<Integer> set = new HashSet<>();

        for(int i:nums1){
            for(int j : nums2){
                if(i == j){
                    set.add(i);
                }
            }
        }
        int[] arr = new int[set.size()];
        int k = 0;
        for(int i : set){
            arr[k]=i;
            k++;
        }

        return arr;
    }
}